Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Commands/Commands.cs

```diff
@@ -130,66 +131,140 @@ public static Command CutActionsOrBindings()
             return (in InputActionsEditorState state) =>
             {
                 CopyPasteHelper.Cut(state);
-                return state.selectionType == SelectionType.Action ?
-                    DeleteAction(state.selectedActionMapIndex, Selectors.GetSelectedAction(state)?.wrappedProperty.FindPropertyRelative(nameof(InputAction.m_Name)).stringValue).Invoke(state)
-                    : DeleteBinding(state.selectedActionMapIndex, state.selectedBindingIndex).Invoke(state);
+                return state.CutActionOrBinding();
             };
         }
 
-        public static Command PasteActionMaps()
+        public static Command PasteActionMaps(IReadOnlyList<IPasteListener> pasteListeners)
         {
             return (in InputActionsEditorState state) =>
             {
-                var lastPastedElement = CopyPasteHelper.PasteActionMapsFromClipboard(state);
+                var newIndex = -99;
+                if (state.GetCutElements() != null && state.GetCutElements().Any())
```

Comment:
Minor: Simplify? `state.GetCutElements()?.Any()` or move that to a getter and instead `state.hasCutElements`

Reply:
good point! fixed in [d04c6ae](https://github.com/Unity-Technologies/InputSystem/pull/1858/commits/d04c6aeed8c317350b8c79fc599f21cae39fbdf5)